### ISSUE TITLE
Missing loss functions?

### ISSUE THREAD

**Issue by listener17** [NoFish-528/encodec-pytorch#5]
Hello!

In Section 3.4 (https://arxiv.org/pdf/2210.13438v1.pdf) loss functions are given.

Which loss functions are still missing in the code?

**Comment by ZhikangNiu** [comment]
you can look the figure1, there is a loss function $l_l$ in quantizer. This loss is still missing in my code.

**Comment by ZhikangNiu** [comment]
And in section 3.4, they have introduced a loss balancer which is also I don't used

**Comment by listener17** [comment]
I see!

**Comment by ZhikangNiu** [comment]
So I close the issue, if you encounter other problems, you can re-open this issue or make a new issue

**Comment by listener17** [comment]
Thanks 🙏
